corehq/apps/locations/templates/locations/manage/locations.html

```diff
@@ -42,9 +42,10 @@
                         data: function (params) {
                             console.log(params);
                             return {
-                                q: params.term,
+                                q: params.term || "",
```

Comment:
What if instead of doing this here, you changed

https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/reports/filters/api.py#L42-42

to

```
self.q = self.request.GET.get('q', '')
```
? Even though this way works, it feels like it's working around the real problem which is that this line in `EmwfOptionsView.get` is giving the `q` param a default value that causes problems (`None`). Changing it there instead of here may fix problems in places that use this as well.